CONTRIBUTING.md

```diff
@@ -136,24 +192,26 @@ As Linguist is a production dependency for GitHub we have a couple of workflow r
 - Anyone with commit rights can merge Pull Requests provided that there is a :+1: from a GitHub staff member.
 - Releases are performed by GitHub staff so we can ensure GitHub.com always stays up to date with the latest release of Linguist and there are no regressions in production.
 
+
 ### Releasing
 
 If you are the current maintainer of this gem:
 
 1. Create a branch for the release: `git checkout -b release-vxx.xx.xx`
 1. Make sure your local dependencies are up to date: `script/bootstrap`
-1. If grammar submodules have not been updated recently, update them: `git submodule update --remote`. If any submodules are updated,
-    1. update the license cache: `script/licensed`
-    1. double check no problems found: `script/licensed status`
-    1. verify and fix any problems identified
-    1. commit all changes: `git commit -a`
+1. If grammar submodules have not been updated recently, update them: `git submodule update --remote`.
+   If any submodules are updated,
+     1. Update the license cache: `script/licensed`
```

Comment:
I wouldn't have used a uppercase letter here; it's the same sentence as `If any submodules are updated,`.

Reply:
😉 Let's concatenate these pieces and see just how that sentence flows.

> 3. "If any submodules are updated, update the license cache, double check no problems found, verify and fix any problems identified, commit all changes"

This should probably end with a colon, rather than a comma: `If any submodules are updated:`

Reply:
> Let's concatenate these pieces and see just how that sentence flows.

Sure, but we still need to give the command to update the license cache (`script/licensed`), right?

Reply:
I mean the sentence should be "If any submodules are updated, update the license cache (`script/licensed`), double check no problems found, verify and fix any problems identified, commit all changes".

Reply:
Probably best if we stick to consistency. 😉 Since the original list didn't terminate each item with a comma, I'm inclined to attribute this to a lazy typist, rather than anything of lexical merit.